scripts/install/setup.sh

```diff
@@ -16,6 +16,21 @@ PARENT_DIR=$PARENT_DIR $PARENT_DIR/spinnaker-for-gcp/scripts/manage/check_git_co
 
 source "$PROPERTIES_FILE"
 
+LOCATION_GKE="--zone $ZONE"
+LOCATION_REDIS="--zone $ZONE"
```

Comment:
In general, the approach is a bit inconsistent with the surrounding scripts.

Should not use command-line args to setup.sh. setup.sh receives all of its properties by sourcing the properties file. setup.sh is idempotent and its approach is to always check for an existing component with the specified name/location before provisioning it.